benchmarks/kmeans/functional_c#/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Immutable;
+using System.Collections.Generic;
+
+namespace functional_c_
+{
+    class Program
+    {
+        public static (double x, double y) computMean(IEnumerable<(double x, double y)> points){
+            var (sumX, sumY) = points
+                            .Aggregate((a, b) => (a.x + b.x, a.y + b.y));
+            var pointsSize = points.Count();
+            return (sumX/pointsSize, sumY/pointsSize);
+        }
+
+        public static (double, double) closest((double x, double y) point, ImmutableList<(double x, double y)> clusters)
+            => clusters.Aggregate((euclideanDist(point, clusters.First()), clusters.First()), (acc, c) => {
+                var dist2 = euclideanDist(point, c);
+                return acc.Item1 < dist2 ? acc : (dist2, c);
+                }).Item2;
+
+        private static double euclideanDist((double x, double y) point, (double x, double y) c)
+            => Math.Sqrt(Math.Pow(point.x - c.x, 2) + Math.Pow(point.y - c.y, 2));
+
+        static void Main(string[] args)
+        {
+            var points = System.IO.File.ReadAllLines("benchmarks/kmeans_concurrent/points.txt")
+                .Select(x => (Convert.ToDouble(x.Split(':')[0]), Convert.ToDouble(x.Split(':')[1])))

```

Comment:
Her splitter du den samme streng to gange i stedet for bare en gang.
Dog tror jeg ikke at det gør så meget lige her.